less/about/preferences.less

```diff
@@ -375,38 +377,6 @@ span.settingsListCopy {
   }
 }
 
-span.browserButton {
-  &.advancedSettings {
-    font-size: 0.9em;
-  }
-
-  &.primaryButton {
-    &.addFunds {
-      font-size: 0.9em;
-      margin-right: 50px;
-    }
-
-    &.clearBrowsingDataButton,
-    &.importNowButton,
-    &.setAsDefaultButton,
-    &.manageAdblockSettings,
-    &.viewExtensionsInfo,
-    &.manageAutofillDataButton {
```

Comment:
These were replaced with `.settingsList > .settingItem + button` and `.settingItem > span + button`. Styling with classes in that way is a cause of inconsistency.